__tests__/app.test.js

```diff
@@ -28,13 +28,44 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
-  // test("404: Responds with message that URL is not found", () => {
-  //   return request(app)
-  //     .get("/app")
-  //     .expect(404)
-  //     .then((res) => {
-  //       console.log(res.body);
-  //       // expect(msg).toBe("Not found");
-  //     });
-  // });
+});
+
+describe("Wrong URL path", () => {
+  test("404: respond with message 'Not found', if the URL path is not valisd", () => {
+    return request(app)
+      .get("/notavalidurl")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
```

Comment:
This is a really good test! I like the `notavalidurl` endpoint as well to make it clear it's not a valid endpoint test! Could consider camelCase or snake-case to make it a little more readable!